iot-hub/Samples/service/JobsSample/Parameters.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using CommandLine;
+
+namespace JobsSample
+{
+    /// <summary>
+    /// Parameters for the application.
+    /// </summary>
+    internal class Parameters
+    {
+        [Option(
+            'c',
+            "HubConnectionString",
+            Required = true,
+            HelpText = "The connection string of the IoT Hub instance to connect to.")]
```

Comment:
Is this a service connection string? If so, which one is required? There are several different built-in types with various permissions.